wp-content/plugins/core/src/Routes/Sample_REST_Route.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Sample REST route.
+ */
+declare( strict_types=1 );
+
+namespace Tribe\Project\Routes;
+
+use Tribe\Libs\Routes\Abstract_Rest_Route;
+
+/**
+ * Class for an example REST route.
+ */
+class Sample_REST_Route extends Abstract_Rest_Route {
```

Comment:
Inconsistent naming styles here with capital `REST`.

Reply:
And as I'm reviewing the tribe-libs PR this is called `Abstract_REST_Route` there, I'll make a note to rename it there.